tools/Azure.Mcp.Tools.Monitor/src/Commands/WebTests/WebTestsCreateOrUpdateCommand.cs

```diff
@@ -0,0 +1,216 @@
+        }
+
+        var timeoutInSeconds = commandResult.GetValueOrDefault(_timeoutInSecondsOption);
+        if (timeoutInSeconds > 120)
+        {
+            response.Status = 400;
+            response.Message = "The timeout cannot be greater than 2 minutes.";
+            return new ValidationResult { IsValid = false };
+        }
+
+        return new ValidationResult { IsValid = true };
+    }
+
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var options = BindOptions(parseResult);
+
+        try
+        {
+            if (!Validate(parseResult.CommandResult, context.Response).IsValid)
+            {
+                return context.Response;
+            }
+
+            var locationsArray = options.Locations!.Split(',', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries);
+            var headersDictionary = options.Headers?
+                .Split(',', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries)
+                .Select(x => x.Split('=', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries))
+                .Where(x => x.Length == 2)
+                .ToDictionary(x => x[0], x => x[1]) ?? new Dictionary<string, string>(0);
+
+            var monitorWebTestService = context.GetService<IMonitorWebTestService>();
+            var webTest = await monitorWebTestService.CreateOrUpdateWebTest(
+                subscription: options.Subscription!,
+                resourceGroup: options.ResourceGroup!,
+                resourceName: options.ResourceName!,
+                appInsightsComponentId: options.AppInsightsComponentId!,
+                location: options.Location!,
+                locations: locationsArray,
+                requestUrl: options.RequestUrl!,
+                webTestName: options.WebTestName,
+                description: options.Description,
+                enabled: options.Enabled,
+                expectedStatusCode: options.ExpectedStatusCode,
+                followRedirects: options.FollowRedirects,
+                frequencyInSeconds: options.FrequencyInSeconds,
+                headers: headersDictionary,
+                httpVerb: options.HttpVerb,
+                ignoreStatusCode: options.IgnoreStatusCode,
+                parseRequests: options.ParseRequests,
+                requestBody: options.RequestBody,
+                retryEnabled: options.RetryEnabled,
+                sslCheck: options.SslCheck,
+                sslLifetimeCheckInDays: options.SslLifetimeCheckInDays,
+                timeoutInSeconds: options.TimeoutInSeconds,
+                tenant: options.Tenant,
+                retryPolicy: options.RetryPolicy);
+
+            context.Response.Results = ResponseResult.Create(
+                new WebTestsCreateCommandResult(webTest),
+                MonitorJsonContext.Default.WebTestsCreateCommandResult);
```

Comment:
```suggestion
            context.Response.Results = ResponseResult.Create(
                new(webTest),
                MonitorJsonContext.Default.WebTestsCreateCommandResult);
```

Prefer `new()` when typing can be inferred